### PR TITLE
Fixed lint errors

### PR DESCRIPTION
**Pull request by manojmalik20**
#### Summary
Fixed lint errors

### CODE REVIEW

**Review by hanzei — CHANGES_REQUESTED** [review]
Would you please downstream the changes in `.golangci.yml` from https://github.com/mattermost/mattermost-plugin-github/pull/590/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9?

**Review by mickmister — APPROVED** [review]

File: server/jobs/job_manager.go [hunk 1]
```diff
@@ -16,26 +16,26 @@ import (
 )
 
 type JobManager struct {
+	env            *mscalendar.Env
+	papi           cluster.JobPluginAPI
 	registeredJobs sync.Map
 	activeJobs     sync.Map
-	env            mscalendar.Env
```
Comment (hanzei) [inline server/jobs/job_manager.go:21]: Why was `env` changed to a pointer?

**Review by hanzei — APPROVED** [review]
I don't think the field ordering changes for structs are needed given that `fieldalignment` is disabled, but I'm fine with them.

File: server/jobs/job_manager.go (hunk 1 above)
Comment (manojmalik20) [reply server/jobs/job_manager.go:21]: The previous `golangci-lint` configuration was giving errors here but now it's not. Do you want me to revert this change?

Comment (hanzei) [reply server/jobs/job_manager.go:21]: Yes, please

Comment (manojmalik20) [reply server/jobs/job_manager.go]: Done.

**Pull request merged**